recipes/dotknot/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: dotknot
+  version: 1.3.2
+
+build:
+  number: 0
+  noarch: python
+
+source:
+  url: http://dotknot.csse.uwa.edu.au/DotKnot_1.3.2.tar.gz
+  md5: caf6dbf3559c699d98b5289983f43b2d
+  patches:
+    - dotknot.patch
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - viennarna ==1.8.5
+
+test:
+  source_files:
+    - sample.fasta
+
+  commands:
+    - dotknot.py sample.fasta
```

Comment:
can you simply create a sample.fasta file on the fly with `echo '>foo\nacgtcgttgtgt' > foo.fasta && dotknot.py foo.fasta`?